tests/InitCommandTest.php

```diff
@@ -818,6 +899,10 @@ public function testRunWithClerkMobileApp(): void
                 'README.md',
                 $this->getFixture('default_readme_with_mobile_app_after_using_renovate.md'),
             ],
+            [
+                base_path('composer.json'),
+                $this->getFixture('composer_after_changes.json', true),
```

Comment:
let's rename fixture to

```suggestion
                $this->getFixture('composer_with_pint_settings.json', true),
```